__tests__/app.test.js

```diff
@@ -67,44 +58,94 @@ describe("/api", () => {
 
 describe("/api/topics", () => {
   describe("GET", () => {
-    test("Returns an object with topics key", () => {
+    test("Responds with topics array of objects with keys: slug, description", () => {
       return request(app)
         .get("/api/topics")
         .expect(200)
         .then(({ body }) => {
-          expect(body.topics).toBeTruthy();
+          const topics = body.topics;
+          expect(topics).toHaveLength(3);
+
+          topics.forEach((topic) => {
+            expect(typeof topic.slug).toBe("string");
+            expect(typeof topic.description).toBe("string");
+          });
         });
     });
+  });
+});
 
-    test("Returns topics array of objects with slug, description keys", () => {
+describe("/api/articles", () => {
+  describe("GET", () => {
+    test("Responds with an array of all articles with only the 8 valid keys", () => {
       return request(app)
-        .get("/api/topics")
+        .get("/api/articles")
         .expect(200)
         .then(({ body }) => {
-          const topics = body.topics;
-          expect(topics).toHaveLength(3);
+          const articles = body.articles;
+          expect(articles).toHaveLength(13);
```

Comment:
Really interesting use of .toHaveLength! I really like it! :)

Reply:
Thanks! :)